webapp/src/note_rhs.tsx

```diff
@@ -0,0 +1,179 @@
+import React, {FC, useEffect, useState} from 'react';
+
+import {getCurrentChannelId} from 'mattermost-redux/selectors/entities/common';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {useDispatch, useSelector} from 'react-redux';
+
+import styled from 'styled-components';
+
+import {updateUpstream, fetchNote} from 'client';
+import {makeSelectNoteValue, setNoteValue} from 'redux_connectors';
+
+// @ts-ignore
+const {formatText, messageHtmlToComponent} = window.PostUtils;
```

Comment:
Is there a reason for not using direct imports?

Reply:
It won't work with how webapp is architected at the moment -- trying to import one component will pull in other webapp dependencies and those can't be imported, it's a real mess. Probably it's a webpack issue that we just haven't invested the time into solving. Hoping common components will solve this.